app/controllers/issue_templates_controller.rb

```diff
@@ -107,25 +118,25 @@ def set_pulldown
     end
 
     if inherit_template
-      inherit_templates = get_inherit_templates(project_ids, @tracker.id)
+      inherit_templates = get_inherit_templates(project_ids, tracker_id)
 
       if inherit_templates.any?
-        inherit_templates.each do |x|
-          group.push([x.title, x.id, { class: 'inherited' }])
-          next unless x.is_default == true
-          default_template = x if project_default_template.blank?
+        inherit_templates.each do |template|
+          group.push([template.title, template.id, { class: 'inherited' }])
+          next unless template.is_default == true
+          default_template = template if !has_project_default_template
```

Comment:
__[rubocop]__
* Favor `unless` over `if` for negative conditions. [:ref](https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)